openthread/Kconfig

```diff
@@ -24,4 +26,79 @@ config OPENTHREAD_LIBRARY_1_1
 
 endchoice
 
+# OpenThread configuration features sets from Nordic Semiconductor
+
+choice
+	prompt "Nordic library features set"
+	default OPENTHREAD_NORDIC_LIBRARY_NONE
+	help
+	  This option selects the Thread stack features set used for building the precompiled libraries.
+
+config OPENTHREAD_NORDIC_LIBRARY_NONE
+	bool "None"
+	help
+	  Do not enable any specific set of OpenThread features.
+config OPENTHREAD_NORDIC_LIBRARY_MASTER
+	bool "Master set"
+	select OPENTHREAD_BORDER_AGENT
+	select OPENTHREAD_BORDER_ROUTER
+	select OPENTHREAD_CHILD_SUPERVISION
+	select OPENTHREAD_COAP
+	select OPENTHREAD_COAPS
+	select OPENTHREAD_COMMISSIONER
+	select OPENTHREAD_DIAG
+	select OPENTHREAD_DNS_CLIENT
+	select OPENTHREAD_DHCP6_SERVER
+	select OPENTHREAD_DHCP6_CLIENT
+	select OPENTHREAD_ECDSA
+	select OPENTHREAD_ENABLE_SERVICE
+	select OPENTHREAD_IP6_FRAGM
+	select OPENTHREAD_JAM_DETECTION
+	select OPENTHREAD_JOINER
+	select OPENTHREAD_MAC_FILTER
+	select OPENTHREAD_MTD_NETDIAG
+	select OPENTHREAD_RAW
+	select OPENTHREAD_SLAAC
+	select OPENTHREAD_SNTP_CLIENT
+	select OPENTHREAD_UDP_FORWARD
+	help
+	  Enable the complete set of OpenThread features.
```

Comment:
```suggestion
	Nordic Semiconductor configuration settings for OpenThread master usage.
	This will enable all features in the OpenThread stack.
```

Reply:
OpenThread has more features that are not enabled in these sets. Better to avoid saying _all_.

Reply:
Good point, thanks for the updated version.